modules/Microsoft.ManagedServices/registrationDefinitions/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,37 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'msrdcom'
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: 'Component Validation - <<namePrefix>>${serviceShort} Subscription assignment'
+    authorizations: [
+      {
+        principalId: 'e87a249c-b53b-4685-94fe-863af522e4ee'
+        principalIdDisplayName: 'ResourceModules-Reader'
+        roleDefinitionId: 'acdd72a7-3385-48ef-bd42-f606fba81ae7'
+      }
+      {
+        principalId: 'e2f126a7-136e-443f-b39f-f73ddfd146b1'
+        principalIdDisplayName: 'ResourceModules-Contributor'
+        roleDefinitionId: 'b24988ac-6180-42a0-ab88-20f7382dd24c'
+      }
+      {
+        principalId: '87813317-fb25-4c76-91fe-783af429d109'
+        principalIdDisplayName: 'ResourceModules-LHManagement'
+        roleDefinitionId: '91c1777a-f3dc-4fae-b103-61d183457e46'
+      }
+    ]
+    managedByTenantId: '195ee85d-2f10-4764-8352-a3c99aa772fb'
```

Comment:
This looks good overall from a conversion-to-new-dependencies-approach point of view, but I really don't see this sustainable going forward. Who's tenant is this and how is this resource working if deployed outside of our test environment? I'd suggest to discuss this more in detail with the broader group.

Reply:
It's Marius's. If you have a suggestion where we can get another tenant from which is more sustainable - let me know :D

Reply:
We have an issue open for more than a year (#307) and I just thought about seizing the opportunity to bring that up again giving the review, before it gets forgotten again.